### PR TITLE
Getting close to an end-to-end working module system.

### PR DESCRIPTION
**Pull request by austenmc**

### CODE REVIEW

File: src/Bootstrap.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Mapping\AnsiQuoteStrategy;
+use Doctrine\ORM\Tools\Setup;
+use Doctrine\ORM\Tools\SchemaTool;
+
+class Bootstrap
+{
+    public static function game(): Game
```
Comment (Vassyli) [inline src/Bootstrap.php]: createGame or bootGame would convey this function better than just "game" which returns a GameObject. By reading only the function signature, one could think that this just returns the current game instance. Instead, it boots the entire game.

File: src/Game.php [hunk 2]
```diff
@@ -21,7 +31,7 @@ public function getEntityManager()
      * Returns the game's event manager.
      * @return EventManager The game's event manager.
      */
-    public function getEventManager()
+    public function events(): EventManager
     {
         return $this->eventManager;
     }
```
Comment (Vassyli) [inline src/Game.php:37]: I liked getEventManager and getEntityManager better. It's easier to see what you have (namely an instance of EventManager / EntitiyManager).
Does db return a EntityManager? The PDO connection? The information needed to connect to the database?

**Comment by austenmc** [comment]
Ok, I'll buy these suggestions.

**Comment by Vassyli** [comment]
But otherwise, looks good to me.

**Pull request merged**